### ISSUE TITLE
[test-runner-commands] Select option plugin is missing from the export list.

### ISSUE THREAD

**Issue by seanchoi-dev** [modernweb-dev/web#2292]
https://github.com/modernweb-dev/web/blob/master/packages/test-runner-commands/plugins.mjs

As I see, selectOptionPlugin is missing from export list which causes an error when I try to use selectOption() on my test.
As I tested the example from documentation - https://modern-web.dev/docs/test-runner/commands/#select-option
I get an error as following: 
```
Error: Error while executing command select-option with payload {"selector":"#testSelect","value":"first"}: Unknown command select-option. Did you install a plugin to handle this command?
```

**Comment by Westbrook** [comment]
Great catch! We'd love to accept a PR correcting this issue.

**Comment by web-padawan** [comment]
Fixed by #2479.
